tools/code_format/check_format.py

```diff
@@ -250,6 +251,15 @@
   "extensions/filters/network/common",
   "extensions/filters/network/common/redis",
 }
+
+UNSORTED_FLAGS = {
+  "envoy.reloadable_features.activate_timers_next_event_loop",
+  "envoy.reloadable_features.check_ocsp_policy",
+  "envoy.reloadable_features.grpc_json_transcoder_adhere_to_buffer_limits",
+  "envoy.reloadable_features.http2_skip_encoding_empty_trailers",
+  "envoy.reloadable_features.upstream_http2_flood_checks",
+  "envoy.reloadable_features.header_map_correctly_coalesce_cookies",
```

Comment:
Would it be possible to fix the ordering in source/common/runtime/runtime_features.cc instead?

For instance, when I remove activate_timers_next_event_loop, this code will complain about add_and_validate_scheme_header being out of order since the real problem is that always_apply_route_header_rules was introduced out of order.

Reply:
I can, but that'll screw up the issue filing for those 6 PRs.  Honestly I'd prefer to revert the entire PR than do that but I'm also OK with reverting.,  Thoughts?
Will fix format if we think it's better than nothing :-)

Reply:
I think there's value in enforcing alpha checks on the reloadable flags list, so I'm ok with this PR getting merged as-is.